packages/atclient/src/atkey.c

```diff
@@ -312,20 +308,6 @@ exit: {
 }
 }
 
-int atclient_atkey_to_atstr(const atclient_atkey atkey, atclient_atstr *atstr) {
```

Comment:
If they are helpful for you (if we finally keep some methods using `atclient_atstr`), why not keeping them as `static`? They wont be visible outside this file.

Before usage in the same `.c` file:
`static int atclient_atkey_from_atstr(atclient_atkey *atkey, const atclient_atstr atstr);`

(same for other methods)

Reply:
They are not helpful. In fact, they are not used anywhere. I made them in case I needed them in the future